sky/shell/shell.cc

```diff
@@ -53,6 +59,10 @@ class NonDiscardableMemoryAllocator : public base::DiscardableMemoryAllocator {
 
 base::LazyInstance<NonDiscardableMemoryAllocator> g_discardable;
 
+void ServiceIsolateHook() {
+  DiagnosticServer::Start();
+}
```

Comment:
We should probably make this conditional on the --non-interactive command line flag, like observatory.